install/upgrade_to_2.13.php

```diff
@@ -37,10 +37,11 @@ class PluginFormcreatorUpgradeTo2_13 {
     */
    public function upgrade(Migration $migration) {
       $this->migration = $migration;
+      $this->migrateEntityConfig();
+      $this->migrateFkToUnsignedInt();
       $this->addFormAnswerTitle();
       $this->defaultValuesForTargets();
       $this->migrateItemtypeInQuestion();
-      $this->fixInconsistency();
```

Comment:
this method fixes inconsistency on 2 columns, but does not sets them unsigned. Dropping and relyong on the logic of this PR + GLPI unsigned FK migration